WTWSMS/history/provinces/676 - Kakheti.txt

```diff
@@ -15,8 +15,12 @@ b_davidgareja = temple
 #b_gremi = castle
 
 # Misc
-culture = armenian
-religion = georgian_orthodox
+culture = juhuro
+religion = tzadikite
```

Comment:
I did so intentionally, because I asked "What is the source for that statement"? That is the relevant question.